content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,162 @@
+
+```
+instances:
+
+  # this is my first instance
+  - url: example.com
+
+  # this is my second instance
+  - url: example_2.com
+```
+
+**metrics** - The list of what is collected from your system by each integration. You can find the metrics for each integration in that integrations `metadata.csv` file. For more information about metrics, see the [Metrics][11] developer page. You can also set up [custom metrics][12], so if the integration doesn't offer a metric out of the box, you can usually add it.
+
+**metadata.csv** - The file that lists and stores the metrics collected by each integration.
+
+**custom check** - If you have a unique system that you want to monitor, or if you're going to expand the metrics already sent by an integration, you can build a [custom check][13] to define and send metrics to Datadog. However, if you want to monitor a generally available application, public service, or an open source project, and we don't have an integration that already exists for it, you should consider [building a new integration][1] instead of a custom check.
+
+**tagging** - [Tags][14] are a way to add customization to metrics so that you can filter and visualize them in the most useful way to you.
+
+**logging** - If the system you are monitoring has logs, you can customize the logs you are sending to Datadog and use our [Logging Management solution][15] to manage and analyze them.
+
+**events** - 
+
+**service check** - 
+
+## Setting up an integration
+
+The Datadog Agent package includes integrations officially supported by Datadog, in [integrations core][2]. To use the integrations in integrations core, download the Datadog agent. Community based integrations are in [integrations extras][3], and to use those, you need to download them individually. For more information on installing or managing these integrations, see the [integrations management guide][16].
+
+### API and Application keys
+
+In order to [install the the Datadog Agent package][17], you need an [API and Application key][18]. You can manage your accounts API and Application keys in the [API Settings page][19] of the UI.
+
+### Installation
+
+If you want to connect with a cloud service provider, navigate to that provider on the [Integrations page][20] for specific instructions on how to connect. For other supported integrations, install the [Datadog Agent][21]. Most integrations are supported on our containerized agents: [Docker][22], and [Kubernetes][23]. After you've downloaded the Agent, go to the [Integrations page][20] section to find specific configuration instructions for individual integrations.
+
+### Configuration
+
+Configurations are specific to [individual integrations][20]. In the `conf.d` folder at the root of your Agent's configuration directory, there is a file named `<INTEGRATIONS>.d` in each integration folder, and a sample `conf.yaml` that lists all available configuration options. 
```

Comment:
The file is named `conf.yaml.example` and is located inside the `<INTEGRATION>.d` folder